roles/gluster-centos/defaults/main.yml

```diff
@@ -1,3 +1 @@
 ---
-# when true, tendrl-api is installed from a source repository
```

Comment:
Why is this file staying in `defaults` directory and is not removed along with the directory itself?